smrf/distribute/air_temp.py

```diff
@@ -67,6 +67,7 @@ def initialize(self, topo, data):
         """
 
         self._logger.debug('Initializing distribute.air_temp')
+        self.date_time = date_time
```

Comment:
If something is an optional parameter to a method, but with a default value, I tend to use this:
```python
def initialize(self, topo, data, **kwargs):
            self.date_time = kwargs.pop('date_time', None)
```

Opens up the ability to add more parameters without having to touch the method signature and calling methods can be found easier, that use the optional parameter.

Reply:
Cool!

Reply:
But along the same lines as the data, it's somewhat tough to know what is expected for the method.

Reply:
Sounds like another PR/ticket to look into details for this.